x/distribution/keeper/params.go

```diff
@@ -2,33 +2,24 @@ package keeper
 
 import (
 	"context"
-
 	"cosmossdk.io/math"
 
 	"github.com/cosmos/cosmos-sdk/x/distribution/types"
 )
 
 // GetParams returns the total set of distribution parameters.
-func (k Keeper) GetParams(ctx context.Context) (params types.Params, err error) {
-	store := k.storeService.OpenKVStore(ctx)
-	bz, err := store.Get(types.ParamsKey)
-	if bz == nil || err != nil {
+func (k Keeper) GetParams(ctx context.Context) (types.Params, error) {
+	params, err := k.Params.Get(ctx)
+	if err != nil {
 		return params, err
 	}
-
-	err = k.cdc.Unmarshal(bz, &params)
-	return params, err
+	return params, nil
 }
 
 // SetParams sets the distribution parameters.
 // CONTRACT: This method performs no validation of the parameters.
 func (k Keeper) SetParams(ctx context.Context, params types.Params) error {
```

Comment:
I think for both `SetParams` and `GetParams`, I would run a usage check, if no module is using distribution's `Get/SetParams` besides distribution itself we can remove the methods `Get/SetParams` and replace usage in distribution to use collections directly.